README.md

```diff
@@ -22,9 +22,20 @@ The scanner is located in its [own repository](https://github.com/google/osv-sca
 This repository contains all the code for running https://osv.dev on GCP. This
 consists of:
 
--   API server (`gcp/api`)
--   Web interface (`gcp/appengine`)
--   Workers for bisection and impact analysis (`docker/worker`)
+| directory       | what |
+|-----------------|------|
+| `deployment/`   | Terraform, Cloud Deploy & App Engine config files <br /> A few Cloud Build config yamls <br /> Old (no longer used?) `api-staging` and `api-test` Cloud Run configs  |
+| `docker/`       | CI docker files (`ci`, `deployment`, `terraform`) <br /> Workers for bisection and impact analysis (`worker`, `importer`, `exporter`, `alias`, `worker-base`) <br /> The determine version `indexer`|
```

Comment:
I have a question. Where should I put gke cron folder? I am migrating some crons from gae to gke, and I am thinking about to place cron under docker (docker/cron/backup) or under gcp (gcp/cron/backup)

Reply:
IMO, putting the GKE stuff under docker is most consistent with the current setup